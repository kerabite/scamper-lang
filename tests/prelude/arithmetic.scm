(+ 1 2 3 4 5)
(- 22 7 2 1 8)
(* 1 2 3 4 5)
(/ 22 2 5 4)
(+ 5)
(- 5)
(* 5)
(/ 5)