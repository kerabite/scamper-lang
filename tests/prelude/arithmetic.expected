15
4
120
0.55
5
-5
5
0.2
